doc/source/serve/production-guide/config.md

```diff
@@ -42,7 +48,7 @@ applications:
     ...
 ```
 
-The file contains `proxy_location`, `http_options`, `grpc_options`, and `applications`.
+The file contains `proxy_location`, `http_options`, `grpc_options`, `logging_config` and `applications`.
```

Comment:
the other fields have explanation, this one doesn't

Reply:
done.